<launch>

  <param name="use_sim_time" value="true"/>
  <node pkg="rosbag" type="play" name="imu_bag_player" args="--clock /home/ymp/catkin_ws/src/Module_Robot01/DataBox/imu_test.bag"/>
  <!-- Samplinger -->
  <arg name="samplinghz" default="10"/>
  <arg name="filename" default="test0.txt"/>
  <node name="imu_samplinger" pkg="mod01_cntl" type="mod01_cntl_imu_node">
    <param name="SamplingTime" value="$(arg samplinghz)" type="double"/>
    <param name="FileName" type="string" value="/home/ymp/catkin_ws/src/Module_Robot01/DataBox/$(arg filename)"/>
  </node>
</launch>